2015/proposals/girdhar-yashasvi.md

```diff
@@ -0,0 +1,254 @@
+It was the first time I worked with an organisation on such a large scale. It was an enthralling experience for me to see my code being used by people around the globe.
+That was the reason I chose Mozilla for my next year internship (2014) . I worked on adding the Text to Speech API to Firefox, which was not present till then.
+
+Here are the links to my projects. The code has been shipped at Benetech and ready to ship at Mozilla.
+http://goo.gl/eMqnTN   
+http://goo.gl/iBwsNE
+
+I have also played with gnome-shell source code and fixed a couple of bugs there. Apart from that, as an active
+member of the Open source development Group of my University, I have been involved in organising various events in
+the campus.
+
+
+###Other Professional Experience:
+
+Research and Development Intern with Works Applications in Tokyo (2014 summer) . I built a web application that
+serves as a search engine on the Test Automation Repository. The entire application was built from scratch in two
+months. The product was deployed at the end of the internship. Technologies used were Java Web API's and
+Bootstrap.
+
+
+##Academic Experience
+
+I am currently pursuing my B.Tech in Computer Science and Engineering at IIIT Hyderabad, India.
+
+To enhance my knowledge on my interests, I have taken up courses such as Machine Learning, Information Retrieval
+and Extraction, Software Engineering, Database Systems and Internals of Application servers.
+Some of my projects are :
+* A scalable and efficient search engine on 40 GB Wikipedia corpus.
+* Predict interestingness of an article using sentiment analysis of tweets.
+* Discover patterns in the given data to find various fusion methods for the purpose of building a machine learning model out of it.
+* A standalone application for a restaurant that included designing a database schema for the restaurant and performing queries on it.
+
+
+##Why this project.
+
+Firstly, the reason of choosing Software Carpentry for my summer project is that I strongly believe in the
+vision of the organisation and wanted a way to get involved with it. I think being a Google Summer of Code intern
+this year is one of the best ways to accomplish that.
+It would be a great learning experience for me to observe how the organisation works and I wish to remain
+connected with it even after my project.
+
+Now, the reason for choosing this project is that I am interested in implmenting what is required in the project
+and I think I have the required experience for it. This project attracts me because it provides me a way to use my
+skill set at an organisational level and if completed successfully, it would be a very helpful tool for Software
+Carpentry in future.
+
+I am confident that if I am offered this opportunity, I will be able to contribute positively. I wish to be in a
+long and creative relationship with Software Carpentry and this project would just be a start to it.
+
+<a name="schema"></a>
```

Comment:
On Wed, Mar 25, 2015 at 02:17:43PM -0700, yashasvi girdhar wrote:
> I have actually used this as an anchor point in the markdown itself
> for pointing to the schema from above sections.

Markdown isn't that smart ;).  It's just blindly converting any links.
As long as the anchors are there in the final product, everything will
work out.  For example, [1] works fine.

As a general comment, your header punctuation isn't very consistent.
You have:

* “Other Professional Experience:” (trailing colon)
* “Academic Experience” (no punctuation)
* “Why this project.” (period)
* “Schema :” (space before a colon)

as a set of consecutive headers :p.  I'd suggest dropping the trailing
punctuation from all your headers, except for headers that are
questions (e.g. “Why this project?”), in which case I'd use trailing
question marks.

[1]: https://github.com/itsyash/gsoc2015/blob/master/2015/proposals/girdhar-yashasvi.md#technical-details